guides/common/modules/con_introduction-to-content-management.adoc

```diff
@@ -3,15 +3,15 @@
 
 In the context of {Project}, _content_ is defined as the software installed on systems.
 This includes, but is not limited to, the base operating system, middleware services, and end-user applications.
-With {ProjectNameX}, you can manage the various types of content for Red Hat Enterprise Linux systems at every stage of the software life cycle.
 
```

Comment:
Unsure about this change. You basically move the empty line. I think this is not intentional. IMHO this would make two paragraphs instead of one as it was before.

Reply:
I might have misunderstood a suggestion. Reverted.